packages/template-tab-navigation-ng/src/_app-common.scss

```diff
@@ -12,8 +27,8 @@
 // Common page class
 .page-content {
     .page-placeholder {
-        color: $page-icon-color;
-        font-size: $page-content-placeholder-font-size;
+        color: $grey;
```

Comment:
Better use const() to get these, as free floating variables would be used just for input.